ydb/docs/en/core/reference/observability/metrics/distributed-storage-performance.md

```diff
@@ -0,0 +1,98 @@
+| Sensor Name           | Description                                                                                                                   | Units             |
+|-----------------------|-------------------------------------------------------------------------------------------------------------------------------|-------------------|
+| `DiskTimeAvailable`   | Available disk time.                                                                                                          | arbitrary units   |
+| `UserDiskCost`        | Total cost of requests a VDisk receives from the DS Proxy.                                                                    | arbitrary units   |
+| `InternalDiskCost`    | Total cost of requests received by a VDisk from another VDisk in the group, for example, as part of the replication process.  | arbitrary units   |
+| `CompactionDiskCost`  | Total cost of requests the VDisk sends as part of the compaction process.                                                     | arbitrary units   |
+| `DefragDiskCost`      | Total cost of requests the VDisk sends as part of the defragmentation process.                                                | arbitrary units   |
+| `ScrubDiskCost`       | Total cost of requests the VDisk sends as part of the scrubbing process.                                                      | arbitrary units   |
+| `BurstDetector_redMs` | The duration in milliseconds during which the Token Bucket was in the underflow state.                                         | ms                |
+
+`DiskTimeAvailable` and the request cost are estimates of available and consumed bandwidth, respectively, and not actually measured time. Therefore, both of these values are measured in arbitrary units that, in terms of their physical meaning, are close to nanoseconds.
+
+Performance metrics are displayed on a [dedicated Grafana dashboard](grafana-dashboards.md#ds-performance).
+
+### Conditions for Distributed Storage guarantees {#requirements}
+
+The {{ ydb-short-name }} distributed storage can ensure low response times only under the following conditions:
+
+1. $DiskTimeAvailable >= UserDiskCost + InternalDiskCost + CompactionDiskCost + DefragDiskCost + ScrubDiskCost$ — The average load does not exceed the maximum allowed.
+2. $BurstDetector_redMs = 0$ — There are no short-term load bursts, which would lead to request queues on handlers.
+
+### Performance metrics configuration
+
+Since the coefficients for the request cost formula were measured on specific physical devices from development clusters, and the performance of other devices may vary, the metrics may require additional adjustments to be used as a source of guarantees for Distributed Storage. Performance metric parameters can be managed via [dynamic cluster configuration](../../../maintenance/manual/dynamic-config.md) and the Immediate Controls mechanism without restarting {{ ydb-short-name }} processes.
```

Comment:
не понятно что такое источник гарантий. Может так?
```suggestion
Since the coefficients for the request cost formula were measured on specific physical devices from development clusters, and the performance of other devices may vary, the metrics may require additional adjustments to be used as a reliable source of information about the distributed storage. Performance metric parameters can be managed via [dynamic cluster configuration](../../../maintenance/manual/dynamic-config.md) and the Immediate Controls mechanism without restarting {{ ydb-short-name }} processes.
```